packages/appeals-service-api/src/repositories/document-metadata-repository.js

```diff
@@ -51,6 +51,21 @@ class DocumentMetadataRepository extends MongoRepository {
 			}
 		}
 	}
+
+	/**
+	 * @param {*} documentMetadata
+	 * @returns
+	 */
```

Comment:
idk how these work exactly but something like this?

```
/**
 * @typedef {Object} DocumentMetadata
 * idk tbh
 */


/**
 * @typedef {Object} UpdateResult
 * @property {number} matchedCount
 * @property {number} modifiedCount
 */

/**
 * @param {DocumentMetadata} documentMetadata
 * @returns UpdateResult
 */
```

Reply:
Obviously forgot about this, have moved the doc type into common and also imported the mongo type